paddle/fluid/framework/ir/mkldnn/elt_act_mkldnn_fuse_pass.cc

```diff
@@ -99,8 +99,8 @@ void ElementwiseActivationOneDNNPass::FuseElementwiseAct(
 
   gpd(graph, handler);
   AddStatis(found_elementwise_activation_count);
-  if ((!Has("disable_logs") ||
-       !Get<bool>("disable_logs") && found_elementwise_activation_count > 0))
+  if (!Has("disable_logs") ||
+      !Get<bool>("disable_logs") && (found_elementwise_activation_count > 0))
```

Comment:
This might still need one more open parenthesis
```suggestion
  if ((!Has("disable_logs") ||
      !Get<bool>("disable_logs") && (found_elementwise_activation_count > 0))
```

Reply:
because I wanted this to be primary condition:
`!Has("disable_logs") || !Get<bool>("disable_logs")`
and this secondary condition:
`found_elementwise_activation_count > 0`

Reply:
Done

Reply:
https://en.cppreference.com/w/cpp/language/operator_precedence
in c++ `>` operator is resolved before `&&` so these brackets are not needed.
But it won't hurt either to put it here.